APR-Core/Config.lua

```diff
@@ -181,6 +181,15 @@ function APR.settings:createBlizzOptions()
                     )
                 end
             },
+            statusButton = {
```

Comment:
all buttons must be on the same line
so move it before resetButton and subtract the width of statusButton from the width of buttonOffset

Reply:
![image](https://github.com/Azeroth-Pilot-Reloaded/azeroth-pilot-reloaded/assets/28711160/6986b8bb-42f0-48aa-937e-177200e31bb6)
![image](https://github.com/Azeroth-Pilot-Reloaded/azeroth-pilot-reloaded/assets/28711160/6efd6de3-e26a-46b0-a710-5f0f24d421bf)

buttonOffset = 0.75